src/main/kotlin/org/rust/ide/spellchecker/StringLiteralTokenizer.kt

```diff
@@ -22,9 +22,8 @@ class StringLiteralTokenizer : EscapeSequenceTokenizer<LeafPsiElement>() {
         fun processTextWithEscapeSequences(element: LeafPsiElement, text: String, consumer: TokenConsumer) {
             val unescapedText = StringBuilder()
             val offsets = IntArray(text.length + 1)
-            CodeInsightUtilCore.parseStringCharacters(text, unescapedText, offsets)
-
-            processTextWithOffsets(element, consumer, unescapedText, offsets, 1)
```

Comment:
@Turbo87 is this analysis correct? Or is there some obscure corner case where this `1` is needed?